### DISCUSSION TITLE
현재에는 모바일에서만 작동가능한것이 맞나요?

### DISCUSSION THREAD

**Discussion by qpqpyo** [kimytsc/covid-rest-vaccine-macro#74] [Q&A]
제가 바뀌고 잘 이해가 안되어서 그런데..ㅠㅠ

이제는 모든 과정을 모바일에서 하는 것이 맞나요?
아래 메뉴얼에 있는 부분들도 PC 크롬이 아닌 모바일에서 다 진행하는게 맞는지 여쭙습니다!.. 

1. [다음 홈페이지](https://m.daum.net/)에 접근하여 `카카오 계정으로 로그인`을 합니다.
   이미 `로그인`이 되어 있다면, 이어서 진행해주세요.
1. [내 정보 확인](https://vaccine.kakao.com/api/v1/user)에 접근합니다.  
   만약, `{"error":"error occurred"}`가 보인다면, 로그아웃 후 다시 로그인을 시도해주세요.  
   `{"user":{"name":"나백신","status":"NORMAL","statusDate":null,"statusOrganization":null},"alarms":[],"reservations":[]}`와 같은 메세지가 보인다면 이어서 진행하시면 됩니다.
1. Chrome 브라우저 주소창에 복사한 소스를 붙여넣고 실행시켜줍니다.  
   주의할 점은, 붙여넣을때 앞에 있는 `javascript:`가 없이 붙여넣어지므로, `javascript:`는 직접 입력한 후 붙여넣으시길 바랍니다.
1. 잔여백신 예약에 성공할 경우, `박수소리`와 Mobile은 `진동`이 추가로 울리도록 설정되어 있습니다.

**Comment by kimytsc** [comment]
카카오의 경우, 현재는 모바일 중에서 카카오톡 채팅에서의 브라우저를 통한 안내(=README의 과정)를 따라하시면 됩니다.

PC는 현재 로그인 연동이 막힌 상태라 그렇습니다.
